### PR TITLE
IdP clarification updates for saml configuration and adding an IdP owned domain

### PR DESCRIPTION
**Pull request by sallylanda6**
Thank you for your contribution to the Platform-Management-Docs repo.

## Describe the changes

Added a screen shot and verbiage to configuring SAML about Platform generated unique IDs. Clarified adding an IdP owned domain.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x ] You have signed the [Axway CLA](https://cla.axway.com/)
* [x ] You have verified the technical accuracy of your change
* [x ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new folder of topics>

### CODE REVIEW

File: content/en/docs/management_guide/configuring_and_managing_identity_providers/managing_domains/adding_a_domain.md [hunk 1]
```diff
@@ -16,15 +16,17 @@ date: 2021-08-12
 | \- | 3 | [Enable the IdP configuration for all domain users](/docs/management_guide/configuring_and_managing_identity_providers/enabling_identity_provider_configuration/enabling_idp_configuration_for_all_domain_users/) |
 | \- | 4 | (_Optional_) [Add a subdomain](/docs/management_guide/configuring_and_managing_identity_providers/managing_domains/adding_a_subdomain/) |
 
-At least one email domain used by the Identity Provider must be added to the org, and domain ownership must be confirmed by a TXT record set on the domain's DNS that matches one that is displayed in the Platform.
+At least one email domain used by the Identity Provider must be added to the org and domain ownership must be confirmed by a TXT record set on the domain's DNS that matches one that is displayed in the Platform.
+
+You must check with your DNS provider documentation about specific configuration.
```
Comment (nboxhallburnett) [inline content/en/docs/management_guide/configuring_and_managing_identity_providers/managing_domains/adding_a_domain.md:21]: Should we perhaps word this `You should check your DNS provider's documentation` or `We recommend checking your DNS provider's documentation`?

File: content/en/docs/management_guide/configuring_and_managing_identity_providers/managing_identity_provider_configuration/configuring_a_saml_v2.0_idp.md [hunk 2]
```diff
@@ -43,7 +43,20 @@ To create a new Identity Provider
...
     * **NameID Policy Format**: Click the field under **Required Claim** to show the format that will be used for the **NameID Policy Format** section in the Identity Provider configuration form.
+
         ![Required claim section](/Images/ad_attribute_map_unique_name_id.png)
+
+        ![Identifier name format](/Images/ad_name_identifier_format.png)
+
+        {{% alert title="Note" color="primary" %}}Not all Identity Providers may show the NameID Policy Format with a readable name. Some IdPs may only provide the SAML `urn` string representation.
+        {{% /alert %}}
+        Below is a map of the displayed options to the equivalent `urn` values.
+        * **Persistent** - urn:oasis:names:tc:SAML:2.0:nameid-format:persistent
```
Comment (nboxhallburnett) [inline content/en/docs/management_guide/configuring_and_managing_identity_providers/managing_identity_provider_configuration/configuring_a_saml_v2.0_idp.md:54]: As they're anything but readable, we probably want to display the urn values as monospace to make them a little more distinguishable

File: content/en/docs/management_guide/configuring_and_managing_identity_providers/managing_identity_provider_configuration/configuring_a_saml_v2.0_idp.md [hunk 3]
```diff
@@ -64,13 +77,13 @@ To create a new Identity Provider
 8. Click **Save**. A Confirmation dialog appears with a message that once the Identity Provider is verified, all users on that domain will be required to log into the Platform with their Identity Provider credentials.
     ![Confirm Identity Provider Configuration dialog](/Images/idp_save_configuration.png)
 
-9. To complete the configuration, you must add the values configured in the Platform identity configuration page to your Identity Provider's configuration.
-    * Copy the **Entity ID**, **Assertion Consumer Service URL**, and optionally the **Post-Logout URL**values individually from the Platform's page manually or by clicking the clipboard icon.
+9. To complete the configuration, you must add the values displayed in the Platform identity configuration page to your Identity Provider's configuration.
```
Comment (nboxhallburnett) [inline content/en/docs/management_guide/configuring_and_managing_identity_providers/managing_identity_provider_configuration/configuring_a_saml_v2.0_idp.md:80]: Thoughts on `displayed on the Platform's configured Identity Provider details page.`? If you like that then we'd want to use that in the addition below as well

File: content/en/docs/management_guide/configuring_and_managing_identity_providers/managing_domains/adding_a_domain.md (hunk 1 above)
Comment (sallylanda6) [reply content/en/docs/management_guide/configuring_and_managing_identity_providers/managing_domains/adding_a_domain.md]: See if the change I made is what you intended.

**Pull request merged**